vespalog/src/main/java/com/yahoo/log/InitializeLog.java

```diff
@@ -0,0 +1,20 @@
+package com.yahoo.log;
+
+/**
+ * Sets up Vespa logging. Call a setup method to set up this.
+ *
+ * @author baldersheim
+ */
+public class InitializeLog {
```

Comment:
Was it intentional to put this class in a PublicApi package?

Reply:
No, that is not necessary. I moved it to the log.impl and annotated that with ExportPackage.
I assume that is necessary to access it from other modules.